### PR TITLE
refactor: split config and organization

### PR DESCRIPTION
**Pull request by MafewTM**
## Description

Implement the standardized configuration and organize the files into proper client-server format

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My code fits the style guidelines.
- [ ] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by dusan01 — APPROVED** [review]

File: client/main.lua [hunk 1]
```diff
@@ -17,19 +19,19 @@ local insideJewelry = false
 local electricalBoxEntity
 
 local function createElectricalBox()
-    electricalBoxEntity = CreateObject(`tr_prop_tr_elecbox_01a`, Config.Electrical.x, Config.Electrical.y, Config.Electrical.z, false, false, false)
+    electricalBoxEntity = CreateObject(`tr_prop_tr_elecbox_01a`, config.electrical.x, config.electrical.y, config.electrical.z, false, false, false)
```
Comment (solareon) [inline client/main.lua:22]: Maybe call this electricalBox to be a bit more descriptive?

File: client/main.lua [hunk 2]
```diff
@@ -17,19 +19,19 @@ local insideJewelry = false
...
     while not DoesEntityExist(electricalBoxEntity) do
         Wait(0)
     end
-    SetEntityHeading(electricalBoxEntity, Config.Electrical.w)
-    if Config.UseTarget then
+    SetEntityHeading(electricalBoxEntity, config.electrical.w)
```
Comment (solareon) [inline client/main.lua:26]: same here for electricalBox

File: client/main.lua [hunk 3]
```diff
@@ -87,21 +89,21 @@ AddEventHandler('qb-jewelery:client:electricalHandler', function()
...
     NetworkAddPedToSynchronisedScene(cache.ped, EnterScene, AnimDictBox, 'enter', 1.5, -4.0, 1, 16, 1148846080, 0)
     NetworkAddEntityToSynchronisedScene(Box, EnterScene, AnimDictBox, 'enter_electric_box', 4.0, -8.0, 1)
-    local LoopingScene = NetworkCreateSynchronisedScene(Config.Electrical.x, Config.Electrical.y, Config.Electrical.z, 0, 0, Config.Electrical.w, 2, false, true, 1065353216, -1, 1.0)
+    local LoopingScene = NetworkCreateSynchronisedScene(config.electrical.x, config.electrical.y, config.electrical.z, 0, 0, config.electrical.w, 2, false, true, 1065353216, -1, 1.0)
     NetworkAddPedToSynchronisedScene(cache.ped, LoopingScene, AnimDictBox, 'loop', 1.5, -4.0, 1, 16, 1148846080, 0)
     NetworkAddEntityToSynchronisedScene(Box, LoopingScene, AnimDictBox, 'loop_electric_box', 4.0, -8.0, 1)
-    local LeavingScene = NetworkCreateSynchronisedScene(Config.Electrical.x, Config.Electrical.y, Config.Electrical.z, 0, 0, Config.Electrical.w, 2, true, false, 1065353216, -1, 1.0)
+    local LeavingScene = NetworkCreateSynchronisedScene(config.electrical.x, config.electrical.y, config.electrical.z, 0, 0, config.electrical.w, 2, true, false, 1065353216, -1, 1.0)
```
Comment (solareon) [inline client/main.lua:98]: same here

File: config/client.lua [hunk 4]
```diff
@@ -0,0 +1,11 @@
+return {
+    timeOut = 2700000,
+    minimumPolice = 2,
+    notEnoughPoliceNotify = true,
+    electrical = vec4(-624.295, -215.22, 52.54, 118.0),
+    fingerprintDropChance = 60,
+    useDrawText = false,
+    useTarget = false,
+    alarmDuration = 240000,
+
```
Comment (solareon) [inline config/client.lua:10]: #extraspace

File: config/server.lua [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        [`weapon_advancedrifle`] = true,
+        [`weapon_specialcarbine`] = true,
+        [`weapon_specialcarbine_mk2`] = true,
+        [`weapon_bullpuprifle`] = true,
+        [`weapon_bullpuprifle_mk2`] = true,
+        [`weapon_compactrifle`] = true,
+        [`weapon_militaryrifle`] = true,
+        [`weapon_heavyrifle`] = true,
+        [`weapon_tacticalrifle`] = true,
+    },
```
Comment (solareon) [inline config/server.lua:42]: Spaces and the trailing comma

File: config/shared.lua [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        [9] = {coords = vec3(-450.84, -69.67, 41.29), isOpened = false, isBusy = false, heading = 310.0},
+        [10] = {coords = vec3(-452.13, -68.11, 41.29), isOpened = false, isBusy = false, heading = 310.0},
+        [11] = {coords = vec3(-453.3, -66.62, 41.29), isOpened = false, isBusy = false, heading = 310.0},
+        [12] = {coords = vec3(-454.63, -65.08, 41.29), isOpened = false, isBusy = false, heading = 310.0},
+        [13] = {coords = vec3(-451.44, -73.17, 41.29), isOpened = false, isBusy = false, heading = 40.0},
+        [14] = {coords = vec3(-452.7, -74.22, 41.29), isOpened = false, isBusy = false, heading = 40.0},
+        [15] = {coords = vec3(-455.05, -73.11, 41.29), isOpened = false, isBusy = false, heading = 310.0},
+        [16] = {coords = vec3(-455.5, -70.56, 41.29), isOpened = false, isBusy = false, heading = 220.0},
+        [17] = {coords = vec3(-454.35, -69.64, 41.29), isOpened = false, isBusy = false, heading = 220.0},]]--
+    },
```
Comment (solareon) [inline config/shared.lua:52]: Trailing comma

**Review by solareon — CHANGES_REQUESTED** [review]

**Pull request merged**